emails/views.py

```diff
@@ -1634,6 +1729,17 @@ def _handle_complaint(message_json: AWS_SNSMessageJSON) -> HttpResponse:
         profile.auto_block_spam = True
         profile.save()
 
+    disable_mask_on_complaint_flag, _ = get_waffle_flag_model().objects.get_or_create(
+        name="disable_mask_on_complaint",
+        defaults={
+            "note": (
+                "MPP-3119: When a Relay user marks an email as spam, disable the mask."
+            )
+        },
+    )
+    if disable_mask_on_complaint_flag.is_active_for_user(user):
```

Comment:
This code should be inside the `for recipient in complained_recipients:` loop. `user` is not guaranteed to be set outside of it.

Reply:
Addressed in https://github.com/mozilla/fx-private-relay/pull/5045/commits/5d2e79df59aa660422e44018406d5793651e26e2

Reply:
Note, to move the `get_waffle_flag_model` higher in the `_handle_complaint` code when this is done.